lib/kepler_web/templates/layout/root.html.heex

```diff
@@ -10,6 +10,7 @@
     <script defer phx-track-static type="text/javascript" src={Routes.static_path(@conn, "/assets/app.js")}></script>
   </head>
   <body>
+    <%= render "_user_menu.html", assigns %>
```

Comment:
Consider putting this in the `<header>`

Reply:
Autogenerated—I figure we're going to nuke the layout of these pages and then spruce them up later—but I moved it anyway.